packages/sources/dxfeed-test/src/config/index.ts

```diff
@@ -0,0 +1,36 @@
+export const customSettings = {
+  API_USERNAME: {
+    description: 'username for dxfeed API',
+    type: 'string',
+    default: '',
```

Comment:
Should this have a default? Or remove the default and make it `required: true`?

Reply:
I'll remove the default but not make it required since it is optional for DP